winml/test/api/LearningModelAPITest.cpp

```diff
@@ -26,6 +26,17 @@ static void CreateModelFromFilePath() {
   WINML_EXPECT_NO_THROW(APITest::LoadModel(L"squeezenet_modifiedforruntimestests.onnx", learningModel));
 }
 
+static void CreateModelFileNotFound() {
+  LearningModel learningModel = nullptr;
+
+  WINML_EXPECT_THROW_SPECIFIC(
+    APITest::LoadModel(L"missing_model.onnx", learningModel);,
+    winrt::hresult_error,
+    [](const winrt::hresult_error& e) -> bool {
+          return e.code() == __HRESULT_FROM_WIN32(ERROR_FILE_NOT_FOUND);
```

Comment:
>ERROR_FILE_NOT_FOUND [](start = 50, length = 20)

[https://microsoft.visualstudio.com/OS/_queries/query/396d6640-e06f-4734-9359-401e7e141379](https://microsoft.visualstudio.com/OS/_queries/query/396d6640-e06f-4734-9359-401e7e141379)
Is this bug resolved?

Reply:
which bug are you referring to, this link is to a whole query

Reply:
Sorry, this one. [https://microsoft.visualstudio.com/OS/_queries/edit/24039612/?triage=true](https://microsoft.visualstudio.com/OS/_queries/edit/24039612/?triage=true)

---
In reply to: [414057460](https://github.com/microsoft/onnxruntime/pull/3661#discussion_r414057460) [](ancestors = 414057460)

Reply:
This PR is unrelated to that bug. This PR addressed if the model file path is not found, not if the model file is corrupt.